@xen-orchestra/lite/src/components/vm/VmsActionsBar.vue

```diff
@@ -27,7 +27,12 @@
     <MenuItem v-tooltip="$t('coming-soon')" :icon="faCamera">
       {{ $t("snapshot") }}
     </MenuItem>
-    <MenuItem v-tooltip="$t('coming-soon')" :icon="faTrashCan">
+    <MenuItem
```

Comment:
Could you extract this item and its logic to `src/components/vm/VmActionItems/VmActionDeleteItem.vue`?